ziti/cmd/ziti/cmd/edge_controller/update_identity.go

```diff
@@ -100,12 +100,12 @@ func runUpdateIdentity(o *updateIdentityOptions) error {
 		change = true
 	}
 
-	if o.Cmd.Flags().Changed("defaultHostingCost") {
+	if o.Cmd.Flags().Changed("default-hosting-cost") {
 		setJSONValue(entityData, o.defaultHostingCost, "defaultHostingCost")
 		change = true
 	}
 
-	if o.Cmd.Flags().Changed("defaultHostingPrecedence") {
+	if o.Cmd.Flags().Changed("default-hosting-precedence") {
```

Comment:
is this a bug or a normalization? should it be called out in the release notes?

Reply:
lol - i searched for the 'new term' not the 'old term' i see it in there now . would be nice to mention the change though imo

Reply:
It's not changing the flag. I had the JSON field name there instead of the flag name. So the UI is unchanged, just unbroken now :)